### PR TITLE
Add example scripts to help with onboarding

### PR DESCRIPTION
**Pull request by n1tranquilla**

### CODE REVIEW

File: scripts/download-blocks.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+MAX_LENGTH=200
+
+OUT_DIR=.
+
+for LENGTH in `seq $MIN_LENGTH $MAX_LENGTH`
+do
+    echo "gs://mina_network_block_data/mainnet-${LENGTH}-*.json" >> "$BLOCKS_DIR"/block_urls
+done 2>/dev/null
+
+gsutil -m cp -n -I "$BLOCKS_DIR" < "$BLOCKS_DIR"/block_urls
```
Comment (jhult) [inline scripts/download-blocks.sh:17]: I think we can/should get rid of `gsutil` and download from our own bucket

See https://github.com/Granola-Team/mina-indexer/issues/211#issuecomment-2010888237

**Review by jhult — DISMISSED** [review]
Also requesting to squash commits.

File: scripts/nuke-database.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+rm -rf ~/.mina-indexer/database
```
[the remark this answers is not in the document]
Comment (n1tranquilla) [reply scripts/nuke-database.sh:3]: Yeah, I mean that would make sense. Although the intent is to use at your own risk.

File: scripts/download-blocks.sh (hunk 1 above)
Comment (n1tranquilla) [reply scripts/download-blocks.sh:17]: That would be cool. Would these scripts be targeted to Granolas or to the broader community as well?

Comment (jhult) [reply scripts/download-blocks.sh:17]: I think targeting the broader community would be a good idea.

File: scripts/download-blocks.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -ex
+
+BLOCKS_DIR="./blocks"
+touch "$BLOCKS_DIR"/block_urls
+
+MIN_LENGTH=1
```
Comment (trevorbernard) [inline scripts/download-blocks.sh:7]: This will fail. Blocks start at 2

File: scripts/download-blocks.sh (hunk 1 above)
Comment (trevorbernard) [reply scripts/download-blocks.sh:17]: That's a different issue and shouldn't block this PR

File: scripts/nuke-database.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (Isaac-DeFrain) [reply scripts/nuke-database.sh:3]: I will approve and we can add this later if needed.

**Review by Isaac-DeFrain — APPROVED** [review]
LGTM

**Review by trevorbernard — APPROVED** [review]

**Pull request merged**